libs/application/templates/secondary-school/src/forms/editForm.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  buildAlertMessageField,
+  buildCustomField,
+  buildForm,
+  buildMultiField,
+  buildSection,
+  buildSubmitField,
+} from '@island.is/application/core'
+import { DefaultEvents, Form, FormModes } from '@island.is/application/types'
+import { userInformationSection } from './secondarySchoolForm/userInformationSection'
+import { schoolSection } from './secondarySchoolForm/schoolSection'
+import { extraInformationSection } from './secondarySchoolForm/extraInformationSection'
+import { Logo } from '../assets/Logo'
+import {
+  conclusionSectionFreshman,
+  conclusionSectionGeneral,
+} from './secondarySchoolForm/conclusionSection'
+import { error, overview } from '../lib/messages'
+import { getEndOfDayUTCDate, getFirstRegistrationEndDate } from '../utils'
+
+// Note: This form is identical to SecondarySchoolForm, except added ABORT action in buildSubmitField and custom field HandleBeforeSubmitInEdit
+export const Edit: Form = buildForm({
+  id: 'SecondarySchoolForm',
+  title: '',
+  logo: Logo,
+  mode: FormModes.DRAFT,
+  renderLastScreenButton: false,
+  renderLastScreenBackButton: true,
+  children: [
+    userInformationSection,
+    schoolSection,
+    extraInformationSection,
+    buildSection({
+      id: 'overviewSection',
+      title: overview.general.sectionTitle,
+      children: [
+        buildMultiField({
+          id: 'overviewMultiField',
+          title: overview.general.pageTitle,
+          description: overview.general.description,
+          children: [
+            buildAlertMessageField({
+              id: 'alertPastRegistrationEndDate',
+              alertType: 'error',
+              title: error.errorPastRegistrationDateTitle,
+              message: error.errorPastRegistrationDateDescription,
+              condition: (answers) => {
+                return (
+                  getEndOfDayUTCDate(getFirstRegistrationEndDate(answers)) <
+                  new Date()
+                )
+              },
+            }),
+            buildCustomField({
+              component: 'Overview',
+              id: 'overview',
+              title: '',
+              description: '',
+            }),
+            buildSubmitField({
+              id: 'submit',
+              placement: 'footer',
+              title: '',
+              refetchApplicationAfterSubmit: (event) => {
+                return event === DefaultEvents.ABORT
+              },
+              actions: [
+                {
+                  event: DefaultEvents.ABORT,
```

Comment:
couldn't this just be the same form with a condition on this action and the custom at the bottom?